src/main/app/case/to-api-format.ts

```diff
@@ -216,6 +217,8 @@ const fields: ToApiConverters = {
   }),
   applicant1IsApplicant2Represented: data => ({
     applicant1IsApplicant2Represented: data.applicant1IsApplicant2Represented,
+    applicant2SolicitorRepresented:
+      data.applicant1IsApplicant2Represented === Applicant2Represented.YES ? YesOrNo.YES : YesOrNo.NO,
```

Comment:
I don't think these changes are needed anymore after one of my recent commits - https://github.com/hmcts/nfdiv-frontend/commit/c523c0309a8424c59f28ce528ae8fce652964d5f

Reply:
Ah ok, I'll checkout master as your commits are merged, and add the 'applicant2SolicitorRepresented' line to to-api-format.ts file, then I'll do a test, just to check the Notice of proceeding letters has the solicitor's address if the respondent is represented.

Reply:
applicant2SolicitorRepresented shouldn't need to be added to the to-api-format since it's already in case.ts
We only add to the to-api-format if the format of the field on the FE is different to the format of it in the case api

Reply:
Hi Lara, ok I see, thanks for the explanation. I also did a test using the latest master build and the Notice of Proceeding letters does contain the Solicitor address. Should I close this PR? I can add a comment to the ticket and assign it to Nicola.